### DISCUSSION TITLE
How do I combine hidden tracks if found

### DISCUSSION THREAD

**Discussion by stephen147** [ghDaYuYu/foo_discogs#94] [Feature Request] [answered]
With this [release ](https://www.discogs.com/release/11099319)I get this in the title of the 12th Hidden track _**Bonus Tracks (La La La (Excuse Me Miss Again))**_

Is it possbile to gather all the hidden track titles and place them in the title tag like below?

**_La La La (Excuse Me Miss Again), Stop, Beware... (Jay-Z Remix) and What They Gonna Do Part II_**

So the format would be like:
**_Track 1, Hidden Track 2 and Hidden Track 3_**

**Comment by ghuDaYuYu** [comment]
Just a brief update on this issue...
I guest that at some point this issue will be addressed, unfortunately not soon. The reasons being that there is work that should be previously done: a full revision/rewrite of big parts of the track parsing algorithms, including quite some amount of effort testing the impact of those changes. The code revisions where drafted a while ago, but could not find the time to fully test them.
The outcome would be better hidden tracks management/detection, extended track header functionality (good for classical music),  improved disk#, track# and vinyl side parsing, etc. Unfortunately, I am still pretty unsure how this can be accomplished, maybe through I different branch of the project if there is enough man/hours available to attempt it.
Cheers!

**Comment by stephen147** [comment]
No worries. It's not a regular enough occurrence to bother me and I can manually do them when I find any. But it would be a nice thing to have fixed.

Thanks again for the hard work you're putting into this. Sorry, I can't be of much help :/

**Comment by ghuDaYuYu** [comment]
Yes, no worries. I might enable an optional algorithm choice, at some point, in the preferences, to be turn on for those stubborn releases/cd collections that refuse to be parsed by the original algorithm.

You're welcome, done a lot of collaborations already so thanks to you too.
Good seeing you!

**Comment by stephen147** [comment]
Good to see you too. Happy new year, even though a month has nearly passed 😁😁😁

That optional thing might be a good way to test it out. I'd be happy to test it for you.

**Comment by ghuDaYuYu** [comment, marked as answer]
Fixed in release [v1.0.18](https://github.com/ghDaYuYu/foo_discogger/releases/tag/)
